packages/components/addon/components/hds/side-nav/list/link.hbs

```diff
@@ -20,9 +20,11 @@
       <FlightIcon class="hds-side-nav__list-item-icon-leading" @name={{@icon}} />
     {{/if}}
 
-    <span class="hds-side-nav__list-item-text hds-typography-body-200 hds-font-weight-medium">
-      {{@text}}
-    </span>
+    {{#if @text}}
```

Comment:
Doesn't text need to be here? It should be used as the value for the `aria-label` on the link if it's not visible text.

Reply:
@MelSumner this is an edge case, in which consumers don't use the `@text` but will add `yield`ed content, so the actual content will be determined by them (at the moment the only case we've seen used it is with a `<time>` element